rtsp.go

```diff
@@ -30,9 +30,15 @@ import (
 )
 
 var family = resource.ModelNamespace("erh").WithFamily("viamrtsp")
-var Model = family.WithModel("rtsp")
+var ModelAgnostic = family.WithModel("rtsp")
 var ModelH264 = family.WithModel("rtsp-h264")
-var Models = []resource.Model{Model, ModelH264}
+var ModelH265 = family.WithModel("rtsp-h265")
+var Models = []resource.Model{ModelAgnostic, ModelH264, ModelH265}
+var modelToCodecMap = map[resource.Model]videoCodec{
+	ModelAgnostic: Unknown,
+	ModelH264:     H264,
+	ModelH265:     H265,
+}
```

Comment:
Worth looking at using a `iota` 
https://yourbasic.org/golang/iota/

Reply:
I think `Unknown` is an iota

Reply:
Yeah good point especially if in the future we want more than a direct mapping in our model <-> codec relationship. Added change in new commit